src/components/AddItem.js

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import { db } from '../lib/firebase';
+
+const AddItem = () => {
+  const [groceryItem, setGroceryItem] = useState('');
+  const [daysToPurchase, setDaysToPurchase] = useState(null);
+
+  const onGroceryItemInputChange = (e) => {
+    setGroceryItem(e.target.value);
+  };
+
+  const onSubmitHandler = (e) => {
+    e.preventDefault();
+    db.collection(localStorage.getItem('token')).add({
+      itemName: groceryItem,
+      daysToPurchase: daysToPurchase,
+      lastPurchasedDate: null,
+    });
+
+    setGroceryItem('');
+    setDaysToPurchase(null);
+  };
+
+  const onRadioInputChange = (e) => {
+    setDaysToPurchase(parseInt(e.target.value));
+  };
+
+  return (
+    <div className="add-item">
+      <form onSubmit={onSubmitHandler}>
+        <label htmlFor="addItem">Item name: </label>
+        <input
+          id="addItem"
+          name="addItem"
+          type="text"
+          value={groceryItem}
+          onChange={onGroceryItemInputChange}
+        />
+        <fieldset border="none">
+          <p>How soon will you buy this again?</p>
+          <input
+            id="soon"
+            value="7"
+            type="radio"
+            name="daysToPurchase"
+            onChange={onRadioInputChange}
+            checked={daysToPurchase === 7}
```

Comment:
I like how you all are using the checked attribute passed down from the database return to set the default. 👏 

At some point we may want to abstract the `===` logic here to a central place that holds all the _Soon -> 14, Kind of soon -> 30_ mappings so that our html doesn't have to be concerned with the logic of days equaling a certain duration. Just an optimization to think about for the future!